src/index.js

```diff
@@ -0,0 +1,19 @@
+import addPart from './function.js';
+
+//scale the snek!
+//define what TYPE of ELEMENT to create
+//start by selecting the buttons from the DOM using querySelectorAll,
```

Comment:
Instead of hard coding a bunch of buttons, can you dynamically create buttons?

For example: 

 loop through an array and create a DOM element (button) for each of the types of fruit or each color of scale in your case. 

the array could look something like this ` const typesOfScales = ['brown', 'golden', 'black']`

and create a DOM element for each one.

Good luck!